packages/ag-charts-website/src/content/docs/charts-radial-column-series/index.mdoc

```diff
@@ -0,0 +1,186 @@
+---
+title: "Radial Column Series"
+enterprise: true
+---
+
+**Radial Column** series (sometimes called *Circular Column chart*) represents a dataset in the form of rectangular columns placed along the polar axis. It is similar to [Nightingale Chart](./charts-nightingale-series).
```

Comment:
The ••Radial Column•• series, also known as the Circular Column chart, displays data using rectangular columns that are positioned along a polar axis. This chart type is similar to the [Nightingale Chart](https://chat.openai.com/charts-nightingale-series).